lobby/src/main/java/org/triplea/lobby/server/EnvironmentVariable.java

```diff
@@ -7,17 +7,7 @@
 /** Class that represent OS environment variable keys with default values. */
 @AllArgsConstructor
 public enum EnvironmentVariable {
-  PORT("3304"),
```

Comment:
DB environment specific variables are useful for http server, so there is a need to share it now. 
These values were essentially moved, see:  https://github.com/triplea-game/triplea/pull/4872/files#diff-2c4b408f0009d57a5b1dc98e642bd13a